src/sentry/models/project.py

```diff
@@ -130,7 +131,9 @@ def __str__(self):
     def next_short_id(self):
         from sentry.models import Counter
 
-        with sentry_sdk.start_span(op="project.next_short_id") as span:
+        with sentry_sdk.start_span(op="project.next_short_id") as span, metrics.timer(
```

Comment:
Why is not span not enough?

Reply:
This is an action item from pm. We want to have this in dd as a metric. atm it's not possible to visualize separate spans in sentry ui.

Reply:
ok, makes sense